.gitignore

```diff
@@ -1,2 +1,4 @@
 /aws-cloud-controller-manager
 /cloudconfig
+
+.vscode/
```

Comment:
Can we remove this? I don't think it's typical to have IDE-specific files in `.gitignore`

Reply:
I can delete this if this is a go specific convention, but do you have any specific concerns? I looked at other projects and they seem to do it, eg https://github.com/kubernetes/kops/blob/master/.gitignore

Reply:
I oftn re-use some form of the .gitignore in kubernetes/kubernetes, which has [plenty of IDE-specific files](https://github.com/kubernetes/kubernetes/blob/master/.gitignore#L7-L34):

```
# Eclipse files
.classpath
.project
.settings/**

# Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
.idea/
*.iml

# Vscode files
.vscode

# Emacs save files
*~
\#*\#
.\#*

# Vim-related files
[._]*.s[a-w][a-z]
[._]s[a-w][a-z]
*.un~
Session.vim
.netrwhist
```

Reply:
TIL :)